src/Chapter2.hs

```diff
@@ -741,7 +762,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains e xs = filter (\x -> elem e x) xs
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function and the lambda function inside as well  🙂 
But your implementation is already great 💯